tutorials/boundary_conditions.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<p>End of case 1:</p><p>"We used this type of boundary condition on all but the east boundary in the example above."</p><p>Case 2:</p><p>The boundary values for the advected quantity will be computed automatically using the same constitutive laws as in the interior.</p><p>End of case 3:</p><p>"We used this type of boundary condition on the east boundary in the example above."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>